src/api/entitlements.js

```diff
@@ -54,13 +58,14 @@ export class Entitlements {
    */
   clone() {
     return new Entitlements(
-        this.service,
-        this.raw,
-        this.entitlements.map(ent => ent.clone()),
-        this.product_,
-        this.ackHandler_,
-        this.isReadyToPay,
-        this.decryptedDocumentKey);
+      this.service,
```

Comment:
should this be 4 chars indentation, since it is a continuation line?

Reply:
This formatting was automatically done by `prettier`